src/core/annotation.js

```diff
@@ -617,14 +623,63 @@ class MarkupAnnotation extends Annotation {
     super(parameters);
     const dict = parameters.dict;
 
-    if (!dict.has('C')) {
-      // Fall back to the default background color.
-      this.data.color = null;
+    this.data.isMarkup = true;
+
+    if (dict.has('IT')) {
+      this.data.intent = dict.get('IT').name;
+    }
+
+    if (dict.has('IRT')) {
+      this.data.inReplyTo = dict.getRaw('IRT').toString();
```

Comment:
Similar to my other comment, you need to actually validate this data before calling `toString()` on it to avoid errors.